internal/util/errors.go

```diff
@@ -40,6 +42,10 @@ var (
 	ErrClusterIDNotSet = errors.New("clusterID must be set")
 	// ErrMissingConfigForMonitor is returned when clusterID is not found for the mon.
 	ErrMissingConfigForMonitor = errors.New("missing configuration of cluster ID for monitor")
+	// ErrRBDGroupNotFound is returned when group name is not found in the cluster on the given pool and/or namespace.
```

Comment:
The `internal/util` package should ideally not have references to RBD, CephFS or NFS. There needs to be a good explanation and advantage to do so. I do not see either at the moment.

Possibly a `internal/rbd/errors` package would be more appropriate, you could collect all RBD errors there? But it really depends on the reason why you introduced this change.